crates/rpc/src/handlers/state.rs

```diff
@@ -51,6 +57,15 @@ impl RandaoResponse {
     }
 }
 
+#[serde_as]
+#[derive(Serialize, Deserialize)]
+struct SyncCommitteeResponse {
+    #[serde(with = "serde_utils::quoted_u64_vec")]
+    pub validators: Vec<u64>,
+    #[serde_as(as = "Vec<Vec<DisplayFromStr>>")]
+    pub validator_aggregates: Vec<Vec<u64>>,
```

Comment:
instead of Vec<Vec<u64>> we can define a type Vec<X>

serde(transparent)
struct X{
   serde with u64 ver
   x: Vec<u64>
}